### ISSUE TITLE
Get rid of Activations enum

### ISSUE THREAD

**Issue by ileasile** [JetBrains/KotlinDL#41]
`Activations` enum duplicates `Activation` interface and makes adding new activation function harder for library contributors and impossible for library users.

Get rid of enum and add convenient "provider" service which stores activation functions objects.

**Comment by zaleslaw** [comment]
A good idea, I will think about it. Thanks

**Comment by zaleslaw** [comment]
I agree with your argument about the importance of customization and it's a part of Keras's spirit too.
I suggest planning these API changes in the next release, 0.4.

We should raise a discussion and ask the community about a few of the proposed API changes.

**Comment by knok16** [comment]
Agree, it can break API, plus it can differ from Keras API, which is one more thing to consider.

**Comment by zaleslaw** [comment]
I suppose we could go far from the basic Keras API in a few moments to be a more Kotlin framework adopting the best Kotlin features
